tests/integration_tests/allennlp_tests/tiny_single_id.py

```diff
@@ -5,7 +5,7 @@
 
 from allennlp.data.token_indexers.token_indexer import IndexedTokenList
 from allennlp.data.token_indexers.token_indexer import TokenIndexer
-from allennlp.data.tokenizers.token import Token
+from allennlp.data.tokenizers.token_class import Token
```

Comment:
`allennlp.data.tokenizers.token` was renamed to `allennlp.tokenizers.token_class` in v1.3.0.
(https://github.com/allenai/allennlp/releases/tag/v1.3.0)

I simply replace `token` with `token_class`, as `tiny_single_id.py` is used only in tests.